x-pack/libbeat/autodiscover/providers/aws/elb/config.go

```diff
@@ -30,8 +30,8 @@ type Config struct {
 
 	// AWS Specific autodiscover fields
 
-	Regions []string `config:"regions" validate:"required"`
-	aws.ConfigAWS
+	Regions   []string      `config:"regions" validate:"required"`
+	AWSConfig aws.ConfigAWS `config:",inline"`
```

Comment:
Was there any problem with this field embedded?

Reply:
Yeah, it wasn't stylistically inline with other uses of the AWS config stuff which keeps it inline.